build/repo.targets

```diff
@@ -8,16 +8,6 @@
     <Message Importance="high" Text="##vso[build.addbuildtag]release-candidate" Condition=" '$(IsFinalBuild)' == 'true' " />
   </Target>
 
-  <Target Name="PublishToProdCon">
-    <MSBuild Projects="$(MSBuildThisFileDirectory)..\eng\ProdConPublish.csproj"
-             Targets="Restore"
-             Properties="$(BuildProperties);_Dummy=Restore" />
-
-    <MSBuild Projects="$(MSBuildThisFileDirectory)..\eng\ProdConPublish.csproj"
-             Targets="Publish"
-             Properties="$(BuildProperties)" />
-  </Target>
-
   <Target Name="GenerateProjectList" DependsOnTargets="ResolveProjects">
```

Comment:
We'll need to find a new place to put this. I don't know what the equivalent of repo.targets is in KoreBuild, but importing via Directory.Build.props is not it. The important thing is that we need to invoke `build.cmd /t:GenerateProjectList`, or something like it.

Reply:
I think in general the Arcade version of this is `Build.proj` - that's restored in a package. 

What I did for similar functionality was to put it in `Tools.props` because that's pulled in by `Build.proj`. However, I don't super understand what the purpose is of these things.